README.md

```diff
@@ -1 +1,169 @@
-# capstone-project-1
\ No newline at end of file
+# startup-society-summit
+It's Microverse program capstone (module-1) project
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Startup society summit</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Startup society summit] <a name="about-project"></a>
+
+**[Startup society summit]** is a static website that represents the website for Startup society summit 2023 event or conference.
+
+## 🛠 Built With <a name="built-with">`HTML & CSS/SCSS` and `JavaScript`</a>
+
+### Tech Stack <a name="tech-stack">`HTML`, `CSS/SCSS`, `JavaScript`</a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/webdesign/htmlcss">HTML-CSS</a></li>
+    <li><a href="https://sass-lang.com/">SCSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="https://nodejs.org/">Nodejs</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Clean and Simple Static Webpages]**
+- **[Only Raw HTML-CSS/SCSS-JS codes]**
+- **[Nodejs for development tools]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Nice on deploying the project. However, based on project requirements, you're meant to also record a 5-minute video describing the keynote points of your project. This is in line with having a well-documented code base. Kindly check the image below for detailed instructions. 
- You can use [Loom](https://www.loom.com/looms/videos) for this

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/225585565-a38149d9-6c2f-4e6e-be0d-51f5cc668b42.png)